content/en/synthetics/browser_tests/_index.md

```diff
@@ -53,6 +53,20 @@ Define the configuration of your browser test.
 
 6. Set the **test frequency**: The intervals vary from every five minutes to once per week. To request one-minute frequency, [contact Support][3].
 
+### Snippets
+
+When setting up a new Synthetic Monitoring browser test, use snippets to automatically fill in your devices and regions, rather than selecting these options manually. The following snippets are available:
+
+* **Large screen check**: Automatically perform your browser tests on a large screen across browsers.
+
+* **Tablet check**: Automatically perform your browser tests on a tablet sized screen across browsers.
+
+* **Large screen check**: Automatically perform your browser tests on a mobile sized screen across browsers.
```

Comment:
```suggestion
* **Mobile screen check**: Automatically perform your browser tests on a mobile sized screen across browsers.
```

Reply:
Typo fix above 🙂 I'm also wondering if it would be possible to cut down on the repetitive wording here, because I think the different screen sizes are fairly self-explanatory? Happy to defer to you, but I wonder if introducing the different screen sizes with a single line might make this list (and the similar one in this PR) easier to scan.

Reply:
good point! I'll fix that typo then see how I can cut down on the redundancy here.